### ISSUE TITLE
`mod_proxy_html` missing from Apache 2.4.x builds 

### ISSUE THREAD

**Issue by nhhollander** [bitnami/containers#41361]
### Name and Version

bitnami/apache 2.4.57

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Create a new Apache container with `docker run bitnami/apache:latest`
2. Insert a `httpd.conf` file that attempts to load the `mod_proxy_html` module
3. Restart the server process

### What is the expected behavior?

The `mod_proxy_html` module should be loaded.

### What do you see instead?

Server terminates with `httpd: Syntax error on line 26 of /opt/bitnami/apache/conf/httpd.conf: Cannot load modules/mod_proxy_html.so into server: /opt/bitnami/apache/modules/mod_proxy_html.so: cannot open shared object file: No such file or directory`

### Additional information

The `mod_proxy_html` module has been an Apache base module since version 2.4[^mod_proxy_html]. Prior to this version, it was an optional third party module offered by WebÞing[^webthing]. The docker official `httpd` image for apache 2.4.57 is built with this module included. This can be verified by running `docker run --entrypoint ls httpd -alh /usr/local/apache2/modules/mod_proxy_html.so`

[^mod_proxy_html]: https://httpd.apache.org/docs/2.4/mod/mod_proxy_html.html
[^webthing]: http://apache.webthing.com/

**Comment by carrodher** [comment]
[Here](https://github.com/bitnami/containers/tree/main/bitnami/apache#extend-this-image) you can see some examples about how to extend this image in order to include some modules, could you try this approach?

**Comment by nhhollander** [comment]
It's definitely possible to extend the build and insert a copy of `mod_proxy_html`, I was mostly interested in why the module was not being included in the Bitnami Apache image after it was made a base module. I wasn't sure if this was an intentional omission or if it was just being left out by your automated build systems.

**Comment by joancafom** [comment]
Hi @nhhollander 

I can't see a good reason why we are not shipping it at a first glance. I will create an internal task for the team to evaluate and add support for it, thanks!

**Comment by ctc-longr** [comment]
I too have this same issue. I've extended my own image to include kerberos SSO support with GSSAPI so I'm ok with modifying it, but this being a base module seems best to get from the same build as the original. Can you comment on the move from on-hold to in-progress and back? Is there a technical or other issue which would prevent this from being made available soon? Alternatively, are there remaster instructions for the bitnami build and packaging steps instead of just extending it in-place on-top?

**Comment by gongomgra** [comment]
Hi @nhhollander, @ctc-longr,

We have just released tag `2.4.58-debian-11-r1` of our Apache container adding the mod-proxy-html module (disabled by default).

```
docker run --rm -it bitnami/apache:2.4.58-debian-11-r1 bash
Unable to find image 'bitnami/apache:2.4.58-debian-11-r1' locally
2.4.58-debian-11-r1: Pulling from bitnami/apache
818eb273082b: Pull complete
Digest: sha256:f80cc016c836f9b4ccae1020344dec6266af9f514b0221b17a28bc2f86a39f75
Status: Downloaded newer image for bitnami/apache:2.4.58-debian-11-r1
apache 12:00:24.07 INFO  ==>
apache 12:00:24.07 INFO  ==> Welcome to the Bitnami apache container
apache 12:00:24.07 INFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers
apache 12:00:24.07 INFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues
apache 12:00:24.08 INFO  ==>

I have no name!@e3f1e424f9d4:/app$ ls -larth /opt/bitnami/apache/modules/mod_proxy_html.so
-rwxr-xr-x 1 root root 47K Nov 16 09:17 /opt/bitnami/apache/modules/mod_proxy_html.so
I have no name!@e3f1e424f9d4:/app$ grep -i "loadmodule.*proxy_html" /opt/bitnami/apache/conf/httpd.conf
#LoadModule proxy_html_module modules/mod_proxy_html.so
```

Hope it helps!
